01 - Getting Started.ipynb

```diff
@@ -24,13 +24,13 @@
     "### `conda` / `mamba`\n",
```

Comment:
<p>This solution uses </p><p> </p><pre class="ql-syntax" spellcheck="false">&nbsp;&nbsp;&nbsp; .group_by("island")
&nbsp;&nbsp;&nbsp; .body_mass_g.max()
</pre><p> </p><p>which is fine, but it's a slightly different syntax of what you just introduced which is </p><p> </p><pre class="ql-syntax" spellcheck="false">.group_by("island").agg(penguins.body_mass_g.max())
</pre><p> </p><p>Maybe you can mention both. It's just a nit pick</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>